instrumentation/instaawssdk/dynamodb_tags.go

```diff
@@ -11,47 +11,52 @@ import (
 )
 
 func extractDynamoDBTags(req *request.Request) (opentracing.Tags, error) {
+	var tags opentracing.Tags
 	switch params := req.Params.(type) {
 	case *dynamodb.CreateTableInput:
-		return opentracing.Tags{
+		tags = opentracing.Tags{
 			dynamodbOp:    "create",
 			dynamodbTable: aws.StringValue(params.TableName),
-		}, nil
+		}
 	case *dynamodb.ListTablesInput:
-		return opentracing.Tags{
+		tags = opentracing.Tags{
 			dynamodbOp: "list",
-		}, nil
+		}
 	case *dynamodb.GetItemInput:
-		return opentracing.Tags{
+		tags = opentracing.Tags{
 			dynamodbOp:    "get",
 			dynamodbTable: aws.StringValue(params.TableName),
-		}, nil
+		}
 	case *dynamodb.PutItemInput:
-		return opentracing.Tags{
+		tags = opentracing.Tags{
 			dynamodbOp:    "put",
 			dynamodbTable: aws.StringValue(params.TableName),
-		}, nil
+		}
 	case *dynamodb.UpdateItemInput:
-		return opentracing.Tags{
+		tags = opentracing.Tags{
 			dynamodbOp:    "update",
 			dynamodbTable: aws.StringValue(params.TableName),
-		}, nil
+		}
 	case *dynamodb.DeleteItemInput:
-		return opentracing.Tags{
+		tags = opentracing.Tags{
 			dynamodbOp:    "delete",
 			dynamodbTable: aws.StringValue(params.TableName),
-		}, nil
+		}
 	case *dynamodb.QueryInput:
-		return opentracing.Tags{
+		tags = opentracing.Tags{
 			dynamodbOp:    "query",
 			dynamodbTable: aws.StringValue(params.TableName),
-		}, nil
+		}
 	case *dynamodb.ScanInput:
-		return opentracing.Tags{
+		tags = opentracing.Tags{
 			dynamodbOp:    "scan",
 			dynamodbTable: aws.StringValue(params.TableName),
-		}, nil
+		}
 	default:
 		return nil, errMethodNotInstrumented
 	}
+
+	tags[dynamodbRegion] = aws.StringValue(req.Config.Region)
```

Comment:
You could extract capturing the `dynamodbTable` tag also from the individual cases and put it here, only omitting it if type is "list tables". Just a thought.

Reply:
+1 for this suggestion, although feel free to leave it to another time if you want, as it does not directly impact the change being done.

Reply:
Good point 👍 

I played with code a little bit, but so far I do not have an idea how to do this without an extra if statement :(. In my opinion, having it doesn't make code easier to read.

Reply:
Yes, that would need an extra if statement. It was really just a thought to reduce the code duplication a bit. But I agree, it is totally a matter of taste, so feel free to ignore the suggestion entirely. 👍

Reply:
👍